### PR TITLE
DM-27541: Update SSObject alert schemas

### PR DESCRIPTION
**Pull request by ebellm**

### CODE REVIEW

File: python/lsst/alert/packet/schema/4/0/sample_data/fakeAlert.avro [hunk 1]
```diff
@@ -0,0 +1 @@
...�u@eV�t�(�?  $�933�B33�B  ���?  �D  @Aff$B  �D  @A                                                                                    ���������     �!�Arw5�� �u@eV�t�(�?  $�933�B33�B  ���?  �D  @Aff$B  �D  @A                                                                                    ���������     �!�Arw5�� �u@eV�t�(�?  $�933�B33�B  ���?  �D  @Aff$B  �D  @A                                                                                       �������w5�� �u@eV�t�(�?  $�9  �!�A�P9��9��@   �P9�P9�P9�P9�P9�                                                                                                                                                                                                                                                                                                                                                                                  ��                                                                                                                                                                                                                                                       z�/�k�p]�>>u͗M
```
Comment (natelust) [inline python/lsst/alert/packet/schema/4/0/sample_data/fakeAlert.avro:1]: is there some weird encoding going on in the file here? Could it have been saved not as ascii or UTF-8?

Comment (natelust) [reply python/lsst/alert/packet/schema/4/0/sample_data/fakeAlert.avro:1]: I'm not sure if this file format supports a newline at the end of the file, but if so can you add one?

**Review by natelust — APPROVED** [review]

File: python/lsst/alert/packet/schema/4/0/sample_data/fakeAlert.avro (hunk 1 above)
Comment (timj) [reply python/lsst/alert/packet/schema/4/0/sample_data/fakeAlert.avro:1]: GitHub ignores .gitattributes when showing diffs and so gets very confused at times by binaries that start with plain text (like FITS).

Comment (ebellm) [reply python/lsst/alert/packet/schema/4/0/sample_data/fakeAlert.avro:1]: I'm afraid that avro's plain text schema + binary blob is just the way it is...

**Pull request merged**